bayes_opt/constraint.py

```diff
@@ -77,7 +76,7 @@ def eval(self, **kwargs: dict):
 
         Parameters
         ----------
-        \*\*kwargs :
+        **kwargs : any
```

Comment:
I remember using the backslashes here because otherwise the docs would not render the double asterisks correctly -- something to keep in mind and to check.

Reply:
I found that a backslashe was causing problems with `D417` and fixed it.
But now that I think about it, `pydocstyle` was fine.
It could be an issue with `ruff`.

I'll try to find another way.

Reply:
Changed `.eval()` to ignore `D417`.